sdk/core/azure-core/inc/http/policy.hpp

```diff
@@ -104,6 +109,9 @@ namespace Azure { namespace Core { namespace Http {
     std::unique_ptr<Response> Send(Context& ctx, Request& request, NextHttpPolicy nextHttpPolicy)
         const override
     {
+      auto uuid = UUID().GetUUIDString();
```

Comment:
BTW, speaking of names, should it be `UUID`, or `Uuid`? Can you check with the guidelines?

For example, we write `HTTP` as `Http` as the namespace and `HttpPolicy` class is not named `HTTPPolicy`.

Reply:
FWIW, in .NET, it is `Guid`, not `GUID`

Reply:
In Go SDK, it's called `HTTP` https://github.com/Azure/azure-storage-blob-go/blob/48358e1de5110852097ebbc11c53581d64d47300/azblob/url_blob.go#L125

Reply:
There are a ton of things you can do with a uuid instance. Create one and had it to another function, write it to a binary file, compare 2 uuids with each other to see if they are equal, etc. This is the right design and mimics other languages, like .Net.

Reply:
Good point on the naming.  As our guidelines are written today this should be Uuid.  @JeffreyRichter are you suggesting we make an exception to the guidelines for this class?

Reply:
I think there is a confusion in this comment thread, I think Jeffrey's comment above was a reply to the Jinming's  comment below:
```
"Well, I also feel it's a little bit weird that we construct a UUID instance and then call GetUUIDString to get the real UUID. What else can one do with the UUID object other than call GetUUIDString and who wants to call GetUUIDString more than once?"
```
, and he is not for or against an exception for the naming rules here. This is my perception, I am not speaking for him, of course.

Reply:
Well, storage service returns `"InvalidHttpVerb"` or `"UnsupportedHttpVerb"` in response error reason phrase, while swagger defines `Content-MD5` and `blob-HTTP-headers` and `GetRangeContentCRC64` so it is also not consistent there.
However, for swagger generator's sake, I would vote for acronyms to be written in all caps. @ahsonkhan  for awareness.